src/transformers/models/bloom/convert_bloom_original_checkpoint_to_pytorch.py

```diff
@@ -79,7 +79,11 @@ def get_dtype_size(dtype):
 
 
 def convert_bloom_checkpoint_to_pytorch(
-    bloom_checkpoint_path, bloom_config_file, pytorch_dump_folder_path, shard_model, pretraining_tp
+    bloom_checkpoint_path,
+    bloom_config_file,
+    pytorch_dump_folder_path,
+    shard_model,
+    pretraining_tp,
```

Comment:
Why this change?

Reply:
applying the `[huggingface formatting](https://huggingface.co/docs/transformers/contributing)` said it's okay, so i guess it's fine?

Reply:
Formatting tests pass on the main branch, so there is no reason for this change.